backend/exchanges/pocket.go

```diff
@@ -145,3 +145,47 @@ func PocketWidgetSignAddress(account accounts.Interface, message string, format
 
 	return addr.EncodeForHumans(), base64.StdEncoding.EncodeToString(sig), nil
 }
+
+// PocketWidgetVerifyAddress allows the user to verify an address for the Pocket Iframe workflow.
+// Input params:
+// 	`account` is the account from which the address is derived, and that will be linked to the Pocket order.
+// 	`address` is the address to be verified. It should be the same address previously returned by
+//		`PocketWidgetSignAddress`. Since this should be the first unused address, this function ranges
+//		among them to retrieve the ID needed for the verification.
+//	`format` is the script type that should be used in the address derivation, as received by the widget
+//		(see https://github.com/pocketbitcoin/request-address#requestaddressv0messagescripttype).
+// 	`aoppBTCScriptTypeMap` is the map used in the AOPP flow to get the `ScriptType` object related to the `format` param.
+func PocketWidgetVerifyAddress(account accounts.Interface, address string, format string, aoppBTCScriptTypeMap map[string]signing.ScriptType) (string, error) {
```

Comment:
seems like the return values are error types, so I'd expect the return value to be only `error`.

You can encode the error code in the error itself.

For consistency, you can then forward it as `errorCode` to the frontend, as we do in most other places (in some places we also use the name `errCode`). If you need an additional err message, forward it as `errorMessage`  (also what we commonly use).

Reply:
Right, thanks. Now should be better.